app/src/main/java/org/mozilla/fenix/utils/ToolbarPopupWindow.kt

```diff
@@ -81,12 +81,14 @@ object ToolbarPopupWindow {
             }
         }
 
-        clipboardUrl?.let { url ->
+        clipboardText?.let { text ->
```

Comment:
It would be nice to add some tests for this in ToolbarPopupWindowTest as well. Can we file a good-first-bug issue for this?